backend/src/mat_znx_dft_ops.rs

```diff
@@ -64,11 +66,28 @@ pub trait MatZnxDftOps<BACKEND: Backend> {
     /// * `res`: the [VecZnxDft] to on which to extract the row of the [MatZnxDft].
     /// * `a`: [MatZnxDft] on which the values are encoded.
     /// * `row_i`: the index of the row to extract.
-    fn vmp_extract_row<R, A>(&self, res: &mut R, a: &A, a_row: usize, a_col_in: usize)
+    fn mat_znx_dft_get_row<R, A>(&self, res: &mut R, a: &A, a_row: usize, a_col_in: usize)
     where
         R: VecZnxDftToMut<FFT64>,
         A: MatZnxToRef<FFT64>;
 
+    /// Multiplies A by (X^{k} - 1) and stores the result on R.
+    fn mat_znx_dft_mul_x_pow_minus_one<R, A>(&self, k: i64, res: &mut R, a: &A, scratch: &mut Scratch)
```

Comment:
is this API used anywhere?

Reply:
plan to add `MatZnxDftPrep` and `MatZnxDft` and this will be available on `MatZnxDft`.